python/ray/train/v2/_internal/execution/worker_group/worker_group.py

```diff
@@ -325,6 +325,17 @@ def _start_impl(
             ]
         )
 
+        workers_info = "\n".join(
+            [
+                f"- (node_id={w.metadata.node_id}, ip={w.metadata.node_ip}, "
```

Comment:
Maybe we should get rid of the node ID. It's kind of long and might be confusing for users, who would be more familiar with IP addresses. Similar to how we don't include it in task/actor logs.
```suggestion
                f"- ip={w.metadata.node_ip}, "
```

Reply:
agreed